app/uk/gov/hmrc/teamsandrepositories/services/PersistingService.scala

```diff
@@ -59,15 +59,17 @@ case class PersistingService @Inject()(
       teamRepos <- teams.foldLeftM(List.empty[TeamRepositories]) { case (acc, team) =>
         dataSource.getTeamRepositories(team).map(_ :: acc)
       }
-      reposWithTeams  = teamRepos.foldLeft(Map.empty[String, GitRepository]) { case (acc, trs) =>
+      reposWithTeams = teamRepos.foldLeft(Map.empty[String, GitRepository]) { case (acc, trs) =>
         trs.repositories.foldLeft(acc) { case (acc, repo) =>
           val r = acc.getOrElse(repo.name, repo)
           acc + (r.name -> r.copy(teams = trs.teamName :: r.teams))
         }
       }
-      _ = logger.info(s"found ${reposWithTeams.values.size} repos")
-      count <- persister.updateRepos(reposWithTeams.values.toSeq)
+      allRepos      <- dataSource.getAllRepositoriesByName()
+      orphanRepos    = (allRepos -- reposWithTeams.keys).values
```

Comment:
CLARIFICATION: Orphan repos don't appear to be used anywhere? (they're calculated by taking the repos with teams away from all repos and then added again to get the repos to persist - couldn't you just say `persister.updateRepos(allRepos)`?

Reply:
No because then we wouldn't be persisting any of the team info.